### ISSUE TITLE
Core room for key structure positioning needs to check structure difficulty

### ISSUE THREAD

**Issue by Turmfalke2** [ReikaKalseki/Reika_Mods_Issues#1700]
![2017-11-25_13 47 09](https://user-images.githubusercontent.com/5174562/33230625-5cfa3a8a-d1e7-11e7-9554-6fae69fe19ae.png)
![2017-11-25_13 47 42](https://user-images.githubusercontent.com/5174562/33230628-617bd942-d1e7-11e7-8ee1-5629ff91dd1c.png)
![2017-11-25_13 47 50](https://user-images.githubusercontent.com/5174562/33230630-671baac6-d1e7-11e7-8b21-734b26fe4c97.png)

It is directly on the opposite.

v19c

**Comment by Turmfalke2** [comment]
Seems to happen everytime, this time the structure spawned in a void rift.
![2017-11-25_14 18 57](https://user-images.githubusercontent.com/5174562/33230885-b4fa0b62-d1eb-11e7-87b2-4d99fa581c53.png)

**Comment by Joshi120** [comment]
This happend to me too but i noticed it only happend when the difficulty was set to 1 this never happend on 3

**Comment by ReikaKalseki** [comment]
> it only happend when the difficulty was set to 1 this never happend on 3

Can someone confirm this?

**Comment by Joshi120** [comment]
Here we go i hope this helps 


![javaw_2017-11-25_23-50-11](https://user-images.githubusercontent.com/5485037/33235616-f0833532-d23b-11e7-9fb1-60c598b50b39.png)
![2017-11-25_23-49-32](https://user-images.githubusercontent.com/5485037/33235617-f09d9d14-d23b-11e7-98d2-a0ddcb7714e1.jpg)
![2017-11-25_23-09-06](https://user-images.githubusercontent.com/5485037/33235618-f0b4d22c-d23b-11e7-9cff-309227dd508f.png)

**Comment by ReikaKalseki** [comment]
Well, this...might...be easy to fix...

**Comment by Turmfalke2** [comment]
Right, I did the set the difficulty to 1 prior to that. Might as well set it back again given that music puzzle is still completely impossible for me.

**Comment by Joshi120** [comment]
the music one is probably the simplest just number the buttons clockwise from 1-4 and look on the circles when the crystals get triggered

**Comment by Turmfalke2** [comment]
I have done that. I even went so far as to record it with obs and play it back frame by frame. I have no idea how to input a chord, because already in the first frame 3 notes play at the same time.

**Comment by Joshi120** [comment]
just pick one per chord you just need one of them
